pkg/solidity-utils/contracts/test/UniswapV2BasePoolMathMock.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+pragma solidity ^0.8.24;
+
+import "@openzeppelin/contracts/utils/math/Math.sol";
+
+import "../math/FixedPoint.sol";
+import "./BasePoolMathMock.sol";
+
+contract UniswapV2BasePoolMathMock is BasePoolMathMock {
```

Comment:
Comment on why we have this? Do we expect to have partner pools that use this invariant?